neutron/assetlist.json

```diff
@@ -1451,6 +1451,117 @@
         "twitter": "https://x.com/ArenaDAO",
         "website": "https://arenadao.org/"
       }
-    }
+    },
+    {
+     "description": "Astrovault AXV",
+      "denom_units": [
+        {
+          "denom": "cw20:neutron10dxyft3nv4vpxh5vrpn0xw8geej8dw3g39g7nqp8mrm307ypssksau29af",
+          "exponent": 0
+        },
+        {
+          "denom": "AXV",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "neutron10dxyft3nv4vpxh5vrpn0xw8geej8dw3g39g7nqp8mrm307ypssksau29af",
+      "base": "cw20:neutron10dxyft3nv4vpxh5vrpn0xw8geej8dw3g39g7nqp8mrm307ypssksau29af",
+      "name": "Astrovault AXV (Neutron)",
+      "display": "AXV",
+      "symbol": "AXV",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/axv.svg"
+        }
+      ],
+      "socials": {
+        "website": "https://astrovault.io/",
+        "twitter": "https://x.com/axvdex"
+      },
+      "coingecko_id": "astrovault"
+    },
+    {
+      "description": "IBC Axelar wbtc-satoshi through axelar-dojo-1 transfer/channel-2",
+      "denom_units": [
+        {
+          "denom": "ibc/DF8722298D192AAB85D86D0462E8166234A6A9A572DD4A2EA7996029DF4DB363",
+          "exponent": 0,
+          "aliases": [
+            "uusdc"
+          ]
+        },
+        {
+          "denom": "axlwbtc",
+          "exponent": 8
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/DF8722298D192AAB85D86D0462E8166234A6A9A572DD4A2EA7996029DF4DB363",
+      "name": "Wrapped Bitcoin (Axelar)",
+      "display": "axlwbtc",
+      "symbol": "axlWBTC",
+      "coingecko_id": "axlwbtc",
```

Comment:
Please remove this CGID. It only needs to be registered to the original asset under Axelar's assetlist.